src/licensedcode/data/licenses/oil-2.0.yml

```diff
@@ -0,0 +1,15 @@
+key: oil-2.0
+short_name: OIL-2.0
+name: Open Innovation License version 2.0
+category: Permissive
+owner: Stark Drones Corporation
+notes: From https://github.com/StarkDrones/OPNLv2/blob/main/.github/LICENSE.md
+homepage_url: https://www.en.patent.aau.dk/open-innovation-licensing/
```

Comment:
Same comment as above for notes and homepage

Reply:
Hey! @pombredanne  I have made the requested changes .